eu.annaaurora.find_billy.Find_Billy.yml

```diff
@@ -0,0 +1,39 @@
+app-id: eu.annaaurora.find_billy.Find_Billy
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+base: org.godotengine.godot.BaseApp
+base-version: '4.3-24.08'
+sdk: org.freedesktop.Sdk
+command: godot-runner
```

Comment:
```suggestion
command: godot-runner
rename-icon: find-billy
```

Reply:
oh, i suppose this resolves the above comment?